Content.Server/SS220/UnembedProjectile/UnembedProjectileSystem.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Server.Projectiles;
+using Content.Shared.Projectiles;
+
+namespace Content.Server.SS220.UnembedProjectile;
+
+public sealed partial class UnembedProjectileSystem : EntitySystem
+{
+    [Dependency] private readonly ProjectileSystem _projectile = default!;
+
+    public int UnembedChildren(EntityUid uid)
+    {
+        int counter = 0;
+
+        if (TryComp(uid, out TransformComponent? transform))
+        {
+            var @enum = transform.ChildEnumerator;
+
+            while (@enum.MoveNext(out var child))
+                if (TryComp<EmbeddableProjectileComponent>(child, out var embeddableComp))
+                    if (embeddableComp.EmbeddedIntoUid == uid)
+                    {
+                        _projectile.UnEmbed(child, embeddableComp);
+                        counter++;
+                    }
```

Comment:
Такое оформление не приму, это слишком, всё таки здесь не одна строчка внутри, а два if можно объединить в один.
```suggestion
            while (@enum.MoveNext(out var child))
            {
                if (TryComp<EmbeddableProjectileComponent>(child, out var embeddableComp)
                    && embeddableComp.EmbeddedIntoUid == uid)
                {
                    _projectile.UnEmbed(child, embeddableComp);
                    counter++;
                }
            }
```